### PR TITLE
Fixed AI seeing more than it should when the Eye is moved to the Core.

### PR DESCRIPTION
**Pull request by xxalpha**
Fixes https://github.com/tgstation/-tg-station/issues/1328

### CODE REVIEW

**Comment by Aranclanos** [comment]
https://github.com/tgstation/-tg-station/pull/5676 is better to accomplish the same goal.

**Comment by GunHog** [comment]
:-1: for the same reasons I posted in the PR Aran mentioned. If you absolutely have to do this, load a camera chunk around the AI itself so it is not blinded when moved or carded. If you were to not do so, I or no AI would volunteer for carding, as being blind really, really sucks.

**Comment by xxalpha** [comment]
Copied all of #5676 and added a portable camera to the AI so it can see. It's not an X-ray camera however, there's no reason for the AI to have X-ray vision.

File: code/modules/mob/living/silicon/ai/freelook/eye.dm [hunk 1]
```diff
@@ -115,10 +93,8 @@
 		src.eyeobj.ai = src
 		src.eyeobj.name = "[src.name] (AI Eye)" // Give it a name
 
-	if(client && client.eye)
-		client.eye = src
-	for(var/datum/camerachunk/c in eyeobj.visibleCameraChunks)
-		c.remove(eyeobj)
+	eyeobj.setLoc(loc)
+	//cameranet.visibility(eyeobj)
```
Comment (RemieRichards) [inline code/modules/mob/living/silicon/ai/freelook/eye.dm]: don't comment it out, remove it.

Comment (xxalpha) [inline code/modules/mob/living/silicon/ai/freelook/eye.dm]: Forgot to clean it up.

**Comment by Anonus** [comment]
One question; Is that camera in the AI's core, or in the AI mob itself? If it's in the AI mob, it probably shouldn't be trackable unless you want malf AIs to be easily found just by using the camera.

Also, this doesn't fix the underlying issue of the AI having xray vision, and it doesn't properly keep players from accessing it; the cancel camera view verb that all mobs have can be used to get the same x-ray vision result as it currently does.
I don't like this, but if you're going to do it, do it correctly.

**Comment by Aranclanos** [comment]
if he took my code then it should be 👌

**Comment by xxalpha** [comment]
Yes it does fix the issue of the AI having x-ray.

**Comment by Anonus** [comment]
Nnno. It only shoves it under a rug, you can still get the x-ray effect by using the cancel camera view verb, and I would bet that you still have the x-ray effect while carded.

**Comment by xxalpha** [comment]
No you can't, I tested it.

**Comment by Aranclanos** [comment]
its my code and my code is always 👌

**Comment by samis** [comment]
I this is merged, add a built-in camera to the AI's core / intelicard so it can see it's surroundings even when Cameraless.

**Comment by Aranclanos** [comment]
you're not paying attention at all because that is included 👌

**Comment by Razharas** [comment]
No idea about the code but if aran says its ok it must be ok

**Comment by Aranclanos** [comment]
👌

**Pull request merged**